### ISSUE TITLE
Suave nuget pulls in half the internet when installed into net461

### ISSUE THREAD

**Issue by 0x53A** [SuaveIO/suave#638]
Create a new net461 project, install suave.

Expected: It adds Suave, and FSharp.Core

Actual: It adds _Everything_.

Why?

![image](https://user-images.githubusercontent.com/4236651/29143843-f26eff28-7d56-11e7-84ad-c0688dab49f0.png)

The package does not have a dependencies block for net40, so it takes the _best matching_ one, which is netstandard1.6.

This is the packages.config:

```
<?xml version="1.0" encoding="utf-8"?>
<packages>
  <package id="FSharp.Core" version="4.1.17" targetFramework="net461" />
  <package id="Microsoft.NETCore.Platforms" version="1.1.0" targetFramework="net461" />
  <package id="Microsoft.Win32.Primitives" version="4.3.0" targetFramework="net461" />
  <package id="NETStandard.Library" version="1.6.1" targetFramework="net461" />
  <package id="Suave" version="2.1.1" targetFramework="net461" />
  <package id="System.AppContext" version="4.3.0" targetFramework="net461" />
  <package id="System.Collections" version="4.3.0" targetFramework="net461" />
  <package id="System.Collections.Concurrent" version="4.3.0" targetFramework="net461" />
  <package id="System.Console" version="4.3.0" targetFramework="net461" />
  <package id="System.Data.Common" version="4.1.0" targetFramework="net461" />
  <package id="System.Diagnostics.Debug" version="4.3.0" targetFramework="net461" />
  <package id="System.Diagnostics.DiagnosticSource" version="4.3.0" targetFramework="net461" />
  <package id="System.Diagnostics.Process" version="4.1.0" targetFramework="net461" />
  <package id="System.Diagnostics.Tools" version="4.3.0" targetFramework="net461" />
  <package id="System.Diagnostics.Tracing" version="4.3.0" targetFramework="net461" />
  <package id="System.Globalization" version="4.3.0" targetFramework="net461" />
  <package id="System.Globalization.Calendars" version="4.3.0" targetFramework="net461" />
  <package id="System.Globalization.Extensions" version="4.3.0" targetFramework="net461" />
  <package id="System.IO" version="4.3.0" targetFramework="net461" />
  <package id="System.IO.Compression" version="4.3.0" targetFramework="net461" />
  <package id="System.IO.Compression.ZipFile" version="4.3.0" targetFramework="net461" />
  <package id="System.IO.FileSystem" version="4.3.0" targetFramework="net461" />
  <package id="System.IO.FileSystem.Primitives" version="4.3.0" targetFramework="net461" />
  <package id="System.Linq" version="4.3.0" targetFramework="net461" />
  <package id="System.Linq.Expressions" version="4.3.0" targetFramework="net461" />
  <package id="System.Linq.Queryable" version="4.0.1" targetFramework="net461" />
  <package id="System.Net.Http" version="4.3.0" targetFramework="net461" />
  <package id="System.Net.Primitives" version="4.3.0" targetFramework="net461" />
  <package id="System.Net.Requests" version="4.0.11" targetFramework="net461" />
  <package id="System.Net.Security" version="4.0.0" targetFramework="net461" />
  <package id="System.Net.Sockets" version="4.3.0" targetFramework="net461" />
  <package id="System.ObjectModel" version="4.3.0" targetFramework="net461" />
  <package id="System.Reflection" version="4.3.0" targetFramework="net461" />
  <package id="System.Reflection.Extensions" version="4.3.0" targetFramework="net461" />
  <package id="System.Reflection.Primitives" version="4.3.0" targetFramework="net461" />
  <package id="System.Resources.ResourceManager" version="4.3.0" targetFramework="net461" />
  <package id="System.Runtime" version="4.3.0" targetFramework="net461" />
  <package id="System.Runtime.Extensions" version="4.3.0" targetFramework="net461" />
  <package id="System.Runtime.Handles" version="4.3.0" targetFramework="net461" />
  <package id="System.Runtime.InteropServices" version="4.3.0" targetFramework="net461" />
  <package id="System.Runtime.InteropServices.RuntimeInformation" version="4.3.0" targetFramework="net461" />
  <package id="System.Runtime.Numerics" version="4.3.0" targetFramework="net461" />
  <package id="System.Runtime.Serialization.Json" version="4.0.2" targetFramework="net461" />
  <package id="System.Security.Claims" version="4.0.1" targetFramework="net461" />
  <package id="System.Security.Cryptography.Algorithms" version="4.3.0" targetFramework="net461" />
  <package id="System.Security.Cryptography.Encoding" version="4.3.0" targetFramework="net461" />
  <package id="System.Security.Cryptography.Primitives" version="4.3.0" targetFramework="net461" />
  <package id="System.Security.Cryptography.X509Certificates" version="4.3.0" targetFramework="net461" />
  <package id="System.Text.Encoding" version="4.3.0" targetFramework="net461" />
  <package id="System.Text.Encoding.Extensions" version="4.3.0" targetFramework="net461" />
  <package id="System.Text.RegularExpressions" version="4.3.0" targetFramework="net461" />
  <package id="System.Threading" version="4.3.0" targetFramework="net461" />
  <package id="System.Threading.Tasks" version="4.3.0" targetFramework="net461" />
  <package id="System.Threading.Tasks.Parallel" version="4.0.1" targetFramework="net461" />
  <package id="System.Threading.Thread" version="4.0.0" targetFramework="net461" />
  <package id="System.Threading.ThreadPool" version="4.0.10" targetFramework="net461" />
  <package id="System.Threading.Timer" version="4.3.0" targetFramework="net461" />
  <package id="System.Xml.ReaderWriter" version="4.3.0" targetFramework="net461" />
  <package id="System.Xml.XDocument" version="4.3.0" targetFramework="net461" />
</packages>
```

Note: this is with VS2017.3 preview.

The relationship between netstandard16 and net461 is ... [complicated](https://docs.microsoft.com/en-us/dotnet/standard/net-standard)

**Comment by haf** [comment]
Nuspec:

```
      <group targetFramework=".NETStandard1.6">
        <dependency id="NETStandard.Library" version="1.6.1" exclude="Build,Analyzers"></dependency>
        <dependency id="FSharp.Core" version="4.1.17" exclude="Build,Analyzers"></dependency>
        <dependency id="System.Data.Common" version="4.1.0" exclude="Build,Analyzers"></dependency>
        <dependency id="System.Diagnostics.Process" version="4.1.0" exclude="Build,Analyzers"></dependency>
        <dependency id="System.Security.Cryptography.Primitives" version="4.3.0" exclude="Build,Analyzers"></dependency>
        <dependency id="System.Net.Security" version="4.0.0" exclude="Build,Analyzers"></dependency>
        <dependency id="System.Globalization.Extensions" version="4.3.0" exclude="Build,Analyzers"></dependency>
        <dependency id="System.Security.Claims" version="4.0.1" exclude="Build,Analyzers"></dependency>
        <dependency id="System.Runtime.Serialization.Json" version="4.0.2" exclude="Build,Analyzers"></dependency>
      </group>
    </dependencies>
```

Could be related to https://github.com/Albacore/albacore/pull/234 and the changes there by @danieljsummers 

For commit 8ba878e

**Comment by haf** [comment]
@0x53A Thanks for your patience. I'm working on a fix for this.

**Comment by haf** [comment]
Fixed in release v2.3.0-beta https://github.com/SuaveIO/suave/releases/tag/v2.3.0-beta
